Tests/test_imagefont.py

```diff
@@ -440,6 +440,18 @@ def test_unicode_pilfont(self):
         with self.assertRaises(UnicodeEncodeError):
             font.getsize(u"’")
 
+    @unittest.skipIf(sys.platform.startswith('win32') and sys.version.startswith('2'),
+                     "requires Python 3.x on Windows")
+    def test_unicode_render(self):
+        # issue #3777
+        text = u"A\u278A\U0001F12B"
+        ttf = ImageFont.truetype("Tests/fonts/NotoSansSymbols-Regular.ttf",
+                                 FONT_SIZE, layout_engine=self.LAYOUT_ENGINE)
+        img = Image.new("RGB", (100, 60))
+        d = ImageDraw.Draw(img)
+        d.text((10, 10), text, font=ttf)
+        self.assert_image_similar_tofile(img, "Tests/images/unicode_extended.png", self.metrics['textsize'])
```

Comment:
Suggestion to allow Lint to pass

```suggestion
        self.assert_image_similar_tofile(
            img, "Tests/images/unicode_extended.png", self.metrics['textsize'])
```

Reply:
I already fixed that, just forgot to commit and push.